lib/lightning_web/components/new_inputs.ex

```diff
@@ -275,7 +277,7 @@ defmodule LightningWeb.Components.NewInputs do
       type="hidden"
       name={@name}
       id={@id}
-      value={Phoenix.HTML.Form.normalize_value(@type, @value)}
+      value={Phoenix.HTML.Form.normalize_value(@type, @value) |> to_string()}
```

Comment:
This got my attention, I can't figure out why it's needed. I mean it's likely harmless, but it's a bit puzzling why this test fails:

```
   test/lightning_web/live/credential_live_test.exs:567
     Assertion with == failed
     code:  assert Enum.count(audit_events) == 2
     left:  1
     right: 2
```

Why is the delete event not created is `nil|true|false` is not cast to a string?

Reply:
Fwiw, it might be a good idea to add `, default: false` to the ProjectCredential delete virtual field - I tried that but didn't solve this specific thing..